src/blocks/PlainTextBlock.tsx

```diff
@@ -1,11 +1,15 @@
 import { colors } from '@hedviginsurance/brand'
 import * as React from 'react'
 import styled from 'react-emotion'
-import { ContentWrapper, SectionWrapper } from '../components/blockHelpers'
+import {
+  ContentWrapper,
+  SectionWrapper,
+  TABLET_BP_DOWN,
+} from '../components/blockHelpers'
 import { BaseBlockProps, MarkdownHtmlComponent } from './BaseBlockProps'
 
 const PlainTextSectionWrapper = styled(SectionWrapper)({
-  paddingBottom: 0,
+  paddingBottom: '0 !important',
```

Comment:
🦀

Reply:
Haha i know, but its either this or overriding all media queries manually